src/main/resources/io/jenkins/plugins/worktile/WTDeployNotifier/help-environmentName.html

```diff
@@ -1,7 +1,4 @@
 <div>
-    部署的环境。
-
-    <strong>
-        如果环境不存在，插件会自动创建。
-    </strong>
+    The name of environment that the code will be deployed to. If the environment does not exist, the plugin will
```

Comment:
The name of environment that the code will be deployed, and will be created for the first time.